nucypher/blockchain/economics.py

```diff
@@ -346,17 +348,24 @@ def __init__(self,
         with localcontext() as ctx:
             ctx.prec = self._precision
 
+            one_year_in_periods = Decimal(ONE_YEAR_IN_HOURS / hours_per_period)
+
             initial_supply = Decimal(initial_supply)
 
             first_phase_supply = Decimal(first_phase_supply)
 
-            first_phase_max_issuance = first_phase_supply / first_phase_duration / 365
+            first_phase_max_issuance = first_phase_supply / first_phase_duration / one_year_in_periods
```

Comment:
If I understand question correctly: switch period is 260th, so with real `one_year_in_periods` we will have reward in 260 period < first phase and 261 > first phase, instead of having it on 260 period exactly. And this lead to more error between model and contract but still small

Reply:
Understood. Small changes like this won't affect the overall incentive mechanism design and its influence, but I suppose they would be pertinent to the proposal from @isaacagudo (where the client independently calculates rewards and the contract is only executed when someone challenges the accuracy/honesty of that computation)

Reply:
Even before calculations was not 100% accurate (how contract works). Anyway need to copy logic from contract to remove such accuracy